src/config/env/prd.json

```
@@ -4,7 +4,7 @@
   "BUY_MANA_URL": "https://decentraland.org/account",
   "BUY_NAME_URL": "https://decentraland.org/builder/claim-name",
   "VESTING_DASHBOARD_URL": "https://decentraland.org/vesting/%23/",
-  "DCL_DATA_API": "https://data.decentraland.vote/",
+  "DCL_DATA_API": "https://raw.githubusercontent.com/Decentraland-DAO/transparency/gh-pages",
```

```suggestion
  "DCL_DATA_API": "https://decentraland-dao.github.io/transparency/",
```
lets use this URL instead